source/codegen/metadata/nixnet/custom_proto.mako

```diff
@@ -0,0 +1,72 @@
+message FlexRayStats {
+    uint32 num_syntax_error_ch_a = 1;
+    uint32 num_syntax_error_ch_b = 2;
+    uint32 num_content_error_ch_a = 3;
+    uint32 num_content_error_ch_b = 4;
+    uint32 num_slot_boundary_violation_ch_a = 5;
+    uint32 num_slot_boundary_violation_ch_b = 6;
+}
+
+message J1939CommState{
+    uint32 pgn = 1;
+    uint32 source_address = 2;
+    uint32 destination_address = 3;
+    uint32 transmit_error = 4;
+    uint32 receive_error = 5;
+}
+
+message CanComm{
+    uint32 comm_state = 1;
+    uint32 transceiver_error = 2;
+    uint32 sleep = 3;
+    uint32 last_error = 4;
+    uint32 transmit_error_counter = 5;
```

Comment:
Nit: Should we use "count" instead of "counter" for the transmit and receive fields like in API's "TxErrCount" and "RxErrCount"?

Reply:
The idea was to keep the field names as close to the names mentioned in the xnet help manual as possible : 
![image](https://user-images.githubusercontent.com/47028785/158217207-c94f1a8d-7faa-4dfd-b840-4e0312fc8cb9.png)